packages/next/build/webpack/plugins/middleware-source-maps-plugin.ts

```diff
@@ -10,7 +10,7 @@ export const getMiddlewareSourceMapPlugins = () => {
   return [
     new webpack.SourceMapDevToolPlugin({
       filename: '[file].map',
-      include: [new RegExp(`^${MIDDLEWARE_FILENAME}.`), /^edge-chunks\//],
+      include: [new RegExp(`${MIDDLEWARE_FILENAME}.`), /^edge-chunks\//],
```

Comment:
Is this going to opt-in anything starting with `middleware`?

Reply:
Anything that contains `middleware.` indeed. 
A lazy way to cover both `middleware.` and `src/middleware.`

But maybe this is not necessary: I'll do more testing with `src/middleware.ts` which should have sourcemaps and involves this webpack plugin.

Reply:
@timneutkens here is the followup PR: https://github.com/vercel/next.js/pull/37582